packages/e2e-tests/specs/editor/various/toolbar-roving-tabindex.test.js

```diff
@@ -101,6 +101,7 @@ describe( 'Toolbar roving tabindex', () => {
 
 	it( 'ensures table block toolbar uses roving tabindex', async () => {
 		await insertBlock( 'Table' );
+		await page.keyboard.press( 'ArrowLeft' );
```

Comment:
Immediately after the table is inserted, the focus is on the input number field (Column Count).
Pressing the up or down key in this state will respect the upper limit of the number (which is a native event), so press the left key in advance to give focus to the block wrapper.

Reply:
This test has been rewritten by Playwright and the file has been removed.
I have commented on it [here](https://github.com/WordPress/gutenberg/pull/43667#discussion_r1199701259) instead.